observer/observer.go

```diff
@@ -0,0 +1,73 @@
+package observer
+
+import (
+	blog "github.com/letsencrypt/boulder/log"
+	// _ are probes imported to trigger init func
+	_ "github.com/letsencrypt/boulder/observer/probes/dns"
+	_ "github.com/letsencrypt/boulder/observer/probes/http"
+	"github.com/prometheus/client_golang/prometheus"
+)
+
+var (
+	statTotalMonitors = prometheus.NewCounterVec(
+		prometheus.CounterOpts{
+			Name: "obs_monitors",
+			Help: "count of configured monitors",
+		},
+		[]string{"probe", "name"},
+	)
+	statTotalErrors = prometheus.NewCounterVec(
+		prometheus.CounterOpts{
+			Name: "obs_errors",
+			Help: "count of errors encountered by all monitors",
+		},
+		[]string{"probe", "name"},
+	)
+	statTotalObservations = prometheus.NewCounterVec(
+		prometheus.CounterOpts{
+			Name: "obs_observations",
+			Help: "count of observations performed by all monitors",
+		},
+		[]string{"probe", "name"},
+	)
+)
+
+// Observer contains the parsed, normalized, and validated configuration
+// describing a collection of monitors and the metrics to be collected
+type Observer struct {
+	Logger   blog.Logger
+	Metric   prometheus.Registerer
+	Monitors []*monitor
+}
+
+// Start registers global metrics and spins off a goroutine for each of
+// the configured monitors
+func (o Observer) Start() {
+	// register metrics
+	o.Metric.MustRegister(statTotalErrors)
+	o.Metric.MustRegister(statTotalMonitors)
+	o.Metric.MustRegister(statTotalObservations)
+
+	// start each monitor
+	for _, mon := range o.Monitors {
+		statTotalMonitors.WithLabelValues(mon.probe, mon.name).Inc()
+		// TODO(@beautifulentropy): track and restart unhealthy goroutines
```

Comment:
Can we at least increment the counter for the number which didn't parse/validate correctly? That way we can easily alert on "the percentage of successful total monitors is too low" instead of "the count of successful total monitors is below [some manual threshol]>"